### PR TITLE
Vep annotation cleanup

### PR DESCRIPTION
**Pull request by bpblanken**

### CODE REVIEW

File: v03_pipeline/lib/annotations/shared_test.py [hunk 1]
```diff
@@ -42,10 +42,8 @@ def test_sorted_transcript_consequences(self) -> None:
                         gene_id='ENSG00000188976',
                         hgvsc='ENST00000327044.6:c.1667C>T',
                         hgvsp='ENSP00000317992.6:p.Ser556Leu',
-                        lof=None,
-                        lof_filter=None,
-                        lof_flags=None,
-                        lof_info='INTRON_END:881781,EXON_END:881925,EXON_START:881782,DE_NOVO_DONOR_MES:-7.36719797135343,DE_NOVO_DONOR_PROB:0.261170618766552,DE_NOVO_DONOR_POS:-138,INTRON_START:881667,DE_NOVO_DONOR_MES_POS:-138,MUTANT_DONOR_MES:4.93863747168278',
+                        is_lof_nagnag=None,
+                        lof_filters=None,
```
Comment (hanars) [inline v03_pipeline/lib/annotations/shared_test.py:46]: could we add a test case for where there are filters?

**Review by hanars — CHANGES_REQUESTED** [review]

File: v03_pipeline/lib/annotations/shared_test.py (hunk 1 above)
Comment (bpblanken) [reply v03_pipeline/lib/annotations/shared_test.py:46]: Yep!

File: luigi_pipeline/lib/hail_vep_runners.py [hunk 2]
```diff
@@ -115,8 +115,8 @@ class HailVEPDummyRunner(HailVEPRunnerBase):
                                                              'hgvsp': 'ENSP00000317992.6:p.Ser556Leu',
                                                              'impact': 'MODERATE',
                                                              'intron': hl.null('str'),
-                                                             'lof': hl.null('str'),
-                                                             'lof_filter': hl.null('str'),
+                                                             'lof': 'LC',
+                                                             'lof_filter': 'END_TRUNC,INCOMPLETE_CDS',
```
Comment (bpblanken) [inline luigi_pipeline/lib/hail_vep_runners.py:119]: hm... actually, I'm not 100% sure this is the right format, gonna go double check.

Comment (bpblanken) [reply luigi_pipeline/lib/hail_vep_runners.py:119]: Ok, I went into `gnomad_v3` and poked around, and I'm fairly confident that this format is correct.

**Review by hanars — APPROVED** [review]

**Pull request merged**